taskcluster/ci/build/kind.yml

```diff
@@ -104,6 +104,18 @@ jobs:
         treeherder:
             symbol: nightly(B)
 
+    android-test:
+        attributes:
+            nightly: false
+        run-on-tasks-for: [github-push]
+        include-nightly-version: true
+        include-shippable-secrets: true
+        run:
+            geckoview-engine: geckoNightly
+            gradle-build-type: fenixNightly
```

Comment:
If we keep the new build type, we'd actually want to use `fennecNightly` (though I expect `fenixNightly` to go away soon anyway).

Reply:
This is confusing - is `fennecNightly` the nightly fenix build or is it `fenixNightly`?

Reply:
`fennecNightly` is the one we want.

For context, they're both nightly builds of Fenix – one has Firefox Preview branding and one has Firefox for Android branding + the migration code. We've been shipping two different nightly channels. This extra flavor should be removed soon though.

Reply:
Ah I see, thanks!

Reply:
For more information, [this document](https://docs.google.com/document/d/1VeAut1_QL_Jvr3xaNj0PADtzmVJq5evbssp8YlYtYVk/edit#) explains what we're going to do with fennec/fenix flavors.